test/framework/kogitoapp.go

```diff
@@ -50,6 +50,10 @@ type KogitoAppDeployment struct {
 	Persistence        bool
 	Events             bool
 	Labels             map[string]string
+	Requests           string
```

Comment:
This will change once https://issues.redhat.com/browse/KOGITO-1597 is merged into master